source/extensions/tracers/common/ot/opentracing_driver_impl.cc

```diff
@@ -102,6 +102,12 @@ void OpenTracingSpan::log(SystemTime timestamp, const std::string& event) {
   finish_options_.log_records.emplace_back(std::move(record));
 }
 
+void OpenTracingSpan::setBaggage(const std::string& key, const std::string& value) {
+  span_->SetBaggageItem(key, value);
+}
+
+std::string OpenTracingSpan::getBaggage(const std::string& key) { return span_->BaggageItem(key); }
```

Comment:
these two methods really just delegate to the underlying Span impl, lightstep's found here

https://github.com/lightstep/lightstep-tracer-cpp/blob/8b7f871242a5d9f514a6eddf0ae51c15749d5ce7/src/tracer/span.cpp#L127-L159